### PR TITLE
Unfreeze personal info page

### PR DESCRIPTION
**Pull request by jacobkagon**
ME-180718046

### CODE REVIEW

**Review by markgoho — CHANGES_REQUESTED** [review]
please add a spec to cover this change, it's possible a cucumber could be augmented to do this

**Review by bill-transue — DISMISSED** [review]
This will fix the personal demographics fields JS validations so that the form still works after validations have been fired.

**Review by markgoho — APPROVED** [review]

File: app/assets/javascripts/demographics_fields.js [hunk 1]
```diff
@@ -307,7 +307,10 @@ var PersonValidations = (function (window, undefined) {
     hidden_requireds = $('[required]').not(':visible');
     $('[required]').not(':visible').removeProp('required');
     this_obj.closest('div').find('button[type="submit"]').trigger('click');
-    this_obj.closest('div').find('button[type="submit"]').prop('disabled',true);
+    this_obj
+    .closest('div')
+    .find('.dependent-disable')
+    .attr('disabled', 'disabled');
```
Comment (markgoho) [inline app/assets/javascripts/demographics_fields.js:313]: should this be `.attr('disabled', true)`?

**Review by markgoho — CHANGES_REQUESTED** [review]

File: app/assets/javascripts/demographics_fields.js (hunk 1 above)
Comment (jacobkagon) [reply app/assets/javascripts/demographics_fields.js:313]: Hey @markgoho just saw this. We identified an issue with the disabled,true where if a user clicked the continue button once, and then had to make changes to the broker registration form, and clicked the button again without reloading the page, the page would remain stuck. This fixes that issue and I verified it locally as well as the cucumber.

Comment (markgoho) [reply app/assets/javascripts/demographics_fields.js:313]: @jacobkagon I'm referring to the syntax of setting a `disabled` attribute on an element, not the strategy of fixing the issue

It looks like this syntax is fine, though -- cf. https://developer.mozilla.org/en-US/docs/Web/API/Element/setAttribute#example

**Review by markgoho — APPROVED** [review]

**Pull request merged**